### ISSUE TITLE
restoreCache shortcut invalid?

### ISSUE THREAD

**Issue by carlin-q-scott** [MicrosoftDocs/azure-devops-yaml-schema#18]
`restoreCache` shortcut for cache task marked as invalid by pipeline validator. Says it's not installed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 387c2e6d-895a-e81c-4f33-4e51e8592e4c
* Version Independent ID: 387c2e6d-895a-e81c-4f33-4e51e8592e4c
* Content: [steps.restoreCache definition](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-restore-cache?view=azure-pipelines)
* Content Source: [content/steps-restore-cache.md](https://github.com/MicrosoftDocs/azure-devops-yaml-schema/blob/main/content/steps-restore-cache.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by steved0x** [comment]
@carlin-q-scott You are correct, I am able to reproduce the issue. This reference is auto generated from the schema, and it looks like there is an error. Let me do a quick test to see if the other related cache shortcut is present, and I will get the documentation fixed. Thank you for letting us know.
